### PR TITLE
Use countdown label as headline for duration of countdown visibility

### PR DESCRIPTION
**Pull request by sookburt**
## What are you doing in this PR?

This ensures that the countdown disappears at the end of it's timespan and the heading of the page is altered to pick up the label associated with the sub-campaign while the countdown is on the page.  It then automatically reverts to the default heading when the countdown disappears (or if there is an empty label).

Also sets the ticker colour scheme based on the figma.

Also changes the ticker colours for the Giving Tuesday campaign.

[**Trello Card**](https://trello.com/c/7RhNsW3L)

## Why are you doing this?

As part of the US end of year campaign, we are using countdown components for a limited period of time.  During the time of the countdown, we would like to change the title of the 3 tier landing page to reflected the limited time till the end of the campaign. 

## How to test

Change the label and start/end dates in the campaign.ts file and run locally using the `?forceCampaign=usEoy2024` flag.

### With label and first countdown and it's label as the title
Screenshot 2024-11-21 at 17 40 46

The countdown disappears automatically and the title switches to the default copy while the page remains in the browser.

### With label and second countdown and it's label as the title
After a refresh, this appears a minute later.
Screenshot 2024-11-21 at 17 43 15

### After countdown has finished the title reverts to the default
Screenshot 2024-11-21 at 17 45 12

### CODE REVIEW

File: support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx [hunk 1]
```diff
@@ -334,6 +337,28 @@ export function ThreeTierLanding({
 		}
 	}, [memoizedCurrentCountdownCampaign]);
 
+	useEffect(() => {
```
Comment (tomrf1) [inline support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:340]: what does this useEffect do?

File: support-frontend/assets/pages/supporter-plus-landing/components/countdown.tsx [hunk 2]
```diff
@@ -106,15 +113,17 @@ export default function Countdown({
 			const isActive =
 				campaign.countdownStartInMillis < now &&
 				campaign.countdownDeadlineInMillis > now;
-			setShowCountdown(isActive);
+			setShow(isActive);
```
Comment (charleycampbell) [inline support-frontend/assets/pages/supporter-plus-landing/components/countdown.tsx:116]: I think `setShowCountdown` might be a clearer name for this function, as it directly reflects its purpose. That said, if there’s a specific reason for the renaming that I might have missed, so feel free to ignore

File: support-frontend/assets/pages/supporter-plus-landing/components/countdown.tsx [hunk 3]
```diff
@@ -88,14 +90,19 @@ const ensureRoundedDoubleDigits = (timeSection: number): string => {
...
 	countdownCampaign: campaign,
 }: CountdownProps): JSX.Element {
 	// one for each timepart to reduce DOM updates where unnecessary.
 	const [seconds, setSeconds] = useState<string>(initialTimePart);
 	const [minutes, setMinutes] = useState<string>(initialTimePart);
 	const [hours, setHours] = useState<string>(initialTimePart);
 	const [days, setDays] = useState<string>(initialTimePart);
-	const [showCountdown, setShowCountdown] = useState<boolean>(false);
+
+	const hideMyself = () => {
```
Comment (charleycampbell) [inline support-frontend/assets/pages/supporter-plus-landing/components/countdown.tsx:103]: Could this be renamed to something a bit clearer? Maybe something like `hideCountdown`

File: support-frontend/stories/landingPage/Countdown.stories.tsx [hunk 4]
```diff
@@ -34,81 +34,90 @@ function Template(args: CountdownProps) {
 	);
 }
 
+function dummy(b: boolean){
```
Comment (charleycampbell) [inline support-frontend/stories/landingPage/Countdown.stories.tsx:37]: can this be removed?

Comment (charleycampbell) [reply support-frontend/stories/landingPage/Countdown.stories.tsx:37]: I see further down it's being passed in so maybe it needs  refactored so it uses the countdown?

File: support-frontend/assets/pages/supporter-plus-landing/components/countdown.tsx (hunk 2 above)
Comment (sookburt) [reply support-frontend/assets/pages/supporter-plus-landing/components/countdown.tsx:116]: I've changed those, thanks!

File: support-frontend/assets/pages/supporter-plus-landing/components/countdown.tsx (hunk 3 above)
Comment (sookburt) [reply support-frontend/assets/pages/supporter-plus-landing/components/countdown.tsx]: I've changed that, thanks!

File: support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx (hunk 1 above)
Comment (tomrf1) [reply support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:340]: I can't see how this useEffect has any effect though, as `getHeadline` just returns a value

Comment (sookburt) [reply support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx]: Removed, thanks

File: support-frontend/stories/landingPage/Countdown.stories.tsx (hunk 4 above)
Comment (sookburt) [reply support-frontend/stories/landingPage/Countdown.stories.tsx]: I've removed the dummy function, thanks

**Pull request merged**